src/core/ext/transport/binder/client/channel_create.h

```diff
@@ -15,17 +15,17 @@
 #ifndef GRPC_CORE_EXT_TRANSPORT_BINDER_CLIENT_CHANNEL_CREATE_H
 #define GRPC_CORE_EXT_TRANSPORT_BINDER_CLIENT_CHANNEL_CREATE_H
 
-#if defined(ANDROID) || defined(__ANDROID__)
-
 #include <grpc/impl/codegen/port_platform.h>
 
 #include <grpc/impl/codegen/grpc_types.h>
 #include <grpc/support/port_platform.h>
 #include <grpcpp/channel.h>
-#include <jni.h>
-
 #include "absl/strings/string_view.h"
 
+#ifdef GPR_ANDROID
```

Comment:
why here and not directly after port_platform.h?

Reply:
Done